packages/@tinacms/fields/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -210,31 +317,41 @@ export const ColorPicker: React.FC<Props> = ({
     )
   }
 
+  const toggleColorPicker = () => {
```

Comment:
Could simplify? 

```
setDisplayColorPicker(!displayColorPicker)
if (!displayColorPicker) {
  //...
}
```

Reply:
I find that less intuitive since you want that conditional to run if the color picker is being displayed. Since it toggles first but the setter is async, you'd use the inverse value (as you have here), but it makes it look like the logic is reversed when it's really not.